parm/aero/berror/aero_diffusionparm.yaml.j2

```diff
@@ -40,11 +40,11 @@ background error:
       - horizontal:
           fixed value: {{ horiz_len }} 
         write:
-          filepath: "{{ DATA }}/berror/diffusion_hz" 
```

Comment:
Do we also need this change in the Var YAML or does the code there copy it to berror?